tools/api_proto_plugin/type_context.py

```diff
@@ -4,8 +4,33 @@
 
 from tools.api_proto_plugin import annotations
 
-# A comment is a (raw comment, annotation map) pair.
-Comment = namedtuple('Comment', ['raw', 'annotations'])
+
+class Comment(object):
+  """Wrapper for comment"""
+
+  def __init__(self, comment, file_level_annotations=None):
+    self.raw = comment
+    self.annotations = annotations.ExtractAnnotations(self.raw, file_level_annotations)
+
+  def transform(self, annotation_xforms):
```

Comment:
As a preference, I'd suggest making this non-mutating and have it return a new `Comment`t. It can be called `getCommentWithTransforms`.